projects/cds/src/profiletag/services/profile-tag.injector.spec.ts

```diff
@@ -106,195 +93,108 @@ describe('ProfileTagInjector', () => {
         },
       ],
     });
-
     profileTagInjector = TestBed.get(ProfileTagInjector);
-    nativeWindow = TestBed.get(WindowRef).nativeWindow;
-  });
 
-  it('should be created', () => {
-    expect(profileTagInjector).toBeTruthy();
-  });
-
-  it('Should first wait for the basesite to be active before adding config parameters to the q array', () => {
-    profileTagInjector.track();
-    const profileTagLoaded$ = profileTagInjector.track();
-    const profileTagLoadedSubcriber = profileTagLoaded$.subscribe();
-    profileTagLoadedSubcriber.unsubscribe();
+    profileTagEventTracker = TestBed.get(ProfileTagEventTracker);
```

Comment:
This should be `profileTagEventTracker = TestBed.get(ProfileTagEventTracker as Type<ProfileTagEventTracker>);`